testing/junit/core/src/main/java/com/adobe/cq/wcm/core/components/testing/MockLanguageManager.java

```diff
@@ -29,6 +29,7 @@
 import com.day.cq.wcm.api.LanguageManager;
 import com.day.cq.wcm.api.Page;
 
+@Deprecated
```

Comment:
IMHO we should just remove the class as there is an alternative with the updated wcm.io AEM Mocks is available and external projects can reference to an older testing package if needed.

Reply:
I'm more than happy to do that :)